site/docs/foundations/color.mdx

```diff
@@ -2,10 +2,35 @@
 title: Color
 layout: DetailTechnical
 data:
-  status: In progress
+
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish our products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts. 
```

Comment:
Even though this was from the old copy, I don't think "our" products makes sense here.

```suggestion
Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts. 
```

Reply:
Also it is worth mentioning that only color can't be used to communicate meaning, should we add colors can be used with text or icons to communicate meaning?